src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx

```diff
@@ -239,21 +244,23 @@ class BreadcrumbsContainer extends React.Component<Props, State> {
     );
   };
 
-  handleFilterBySearchTerm = (value = '') => {
-    const {filteredByFilter} = this.state;
+  handleFilterBySearchTerm = (value: string) => {
+    const {filteredByCustomSearch} = this.state;
 
     const searchTerm = value.toLocaleLowerCase();
 
-    const filteredBreadcrumbs = filteredByFilter.filter(
-      item =>
-        !!['category', 'message', 'level', 'timestamp'].find(prop => {
-          const searchValue = item[prop];
-          if (searchValue) {
-            return searchValue.toLowerCase().indexOf(searchTerm) !== -1;
-          }
+    const filteredBreadcrumbs = filteredByCustomSearch.filter(obj => {
+      return Object.keys(
```

Comment:
nit but we could omit the curly braces and return statement

Reply:
true.